datastream-testcommon/src/main/java/com/linkedin/datastream/connectors/DummyBootstrapConnector.java

```diff
@@ -24,7 +24,7 @@
 
   public static final String CONNECTOR_NAME = "DummyConnectorBootstrap";
 
-  private final HashMap<String, String> _config;
+  protected final HashMap<String, String> _config;
```

Comment:
Not your fault super unimportant feel free to ignore nit: replace `HashMap` with `Map`.